cmd/timescale-prometheus/main.go

```diff
@@ -329,6 +340,54 @@ func parseFlags() (*config, error) {
 	return cfg, nil
 }
 
+var errTimedOut = fmt.Errorf("timed out waiting to connect to DB")
```

Comment:
Since `errTimeOut` is just used once, it will be better to declare these as constants at the top. However, an error in the top of a main file might look odd, so maybe, functions related to wait can be shifted to `utils` and the error being declared there.

Reply:
I think putting it at top of file is fine